src/server/svr_func.c

```diff
@@ -242,6 +244,7 @@ extern enum failover_state are_we_primary(void);
 extern long  svr_history_enable;
 extern long  svr_history_duration;
 extern void  svr_clean_job_history(struct work_task *);
+extern long long svr_max_job_sequence_id;
```

Comment:
Why can't we define it in a header file instead of externing from multiple files?